price_pusher/src/injective/injective.ts

```diff
@@ -99,14 +100,44 @@ export class InjectivePricePusher implements IPricePusher {
     return this.wallet.toBech32();
   }
 
-  private async signAndBroadcastMsg(
-    msg: Msgs,
-    fee = DEFAULT_STD_FEE
-  ): Promise<TxResponse> {
+  private async signAndBroadcastMsg(msg: Msgs): Promise<TxResponse> {
     const chainGrpcAuthApi = new ChainGrpcAuthApi(this.grpcEndpoint);
     const account = await chainGrpcAuthApi.fetchAccount(
       this.injectiveAddress()
     );
+    const { txRaw: simulateTxRaw } = createTransactionFromMsg({
+      sequence: account.baseAccount.sequence,
+      accountNumber: account.baseAccount.accountNumber,
+      message: msg,
+      chainId: "injective-888",
```

Comment:
you may want to make the chainId, the gas price, and the extra_gas_multiplier members of the class (which you can initialize to the defaults in the constructor). That would make the code more flexible in case someone wants to use different values later.